new-lamassu-admin/src/components/fake-table/Table.js

```diff
@@ -108,13 +111,10 @@ const Th = ({ children, ...props }) => {
 const Tr = ({ error, errorMessage, children, className }) => {
   const classes = useStyles()
   const cardClasses = { root: classes.cardContentRoot }
-  const classNames = {
-    [classes.trError]: error
-  }
```

Comment:
Why did this got removed? This is probably being used by other screens. 

You can do the following:

```
const classNames = {
  [classes.trError]: error,
  [classes.card]: true,
  className
}
```

Reply:
Also this has conflicts now, do rebase after doing the changes.